donate/payments/views.py

```diff
@@ -798,15 +801,18 @@ def form_valid(self, form, send_data_to_basket=True):
                 )
 
         elif send_data_to_basket:
-            data['source_url'] = self.request.build_absolute_uri()
-            data['lang'] = self.request.LANGUAGE_CODE
-            queue.enqueue(send_newsletter_subscription_to_basket, data)
-            queue_ga_event(self.request, ['send', 'event', {
-                    'eventCategory': 'Signup',
-                    'eventAction': 'Submitted the Form',
-                    'eventLabel': 'Email',
-                }
-            ])
+            if settings.DONATE_NEWSLETTER_SUBSCRIBE_METHOD.lower() == 'basket':
```

Comment:
if we're setting the value as `BASKET` in the env, and we default to `BASKET` in settings, let's compare to that, not to the lowercase string.

Reply:
Done